Snowflake.Data.Tests/IntegrationTests/MaxLobSizeIT.cs

```diff
@@ -388,6 +386,8 @@ private void AlterSessionSettings(SnowflakeDbConnection conn)
                 //// Alter session max lob
                 //command.CommandText = "ALTER SESSION SET FEATURE_INCREASED_MAX_LOB_SIZE_IN_MEMORY = 'ENABLED'";
                 //command.ExecuteNonQuery();
+                //command.CommandText = "alter session set ALLOW_LARGE_LOBS_IN_EXTERNAL_SCAN = true";
```

Comment:
why is it commented? It would be uncommented later?

Reply:
I added this in case if someone would like to test this for increased max LOB size. Without this change it won't work. Same goes for existing comment above.